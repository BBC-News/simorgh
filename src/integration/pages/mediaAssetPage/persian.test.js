/**
 * @service persian
 * @pathname /persian/iran-23231114
 */

import runCrossPlatformTests from './crossPlatformTests';
import runAmpTests from './ampTests';
import runCanonicalTests from './canonicalTests';

describe('Media Asset Page', () => {
  describe(platform, () => {
    runCrossPlatformTests();
  });

  if (platform === 'amp') {
    describe(platform, () => {
      runAmpTests();
    });
  }

  if (platform === 'canonical') {
    describe(platform, () => {
      runCanonicalTests();
    });
  }
});
