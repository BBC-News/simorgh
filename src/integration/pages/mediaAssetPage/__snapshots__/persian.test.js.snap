// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media Asset Page amp A11y I can see the skip to content link 1`] = `"مشاهده محتوا"`;

exports[`Media Asset Page amp Core AMP AMP Consent config 1`] = `"{\\"consents\\":{\\"user-consent\\":{\\"promptIfUnknownForGeoGroup\\":\\"eea\\",\\"promptUI\\":\\"consent-prompt\\"}}}"`;

exports[`Media Asset Page amp Core AMP AMP Geo config 1`] = `"{\\"ISOCountryGroups\\":{\\"eea\\":[\\"at\\",\\"ax\\",\\"be\\",\\"bg\\",\\"bl\\",\\"cy\\",\\"cz\\",\\"de\\",\\"dk\\",\\"ea\\",\\"ee\\",\\"es\\",\\"fi\\",\\"fr\\",\\"gb\\",\\"gf\\",\\"gg\\",\\"gi\\",\\"gp\\",\\"gr\\",\\"hr\\",\\"hu\\",\\"ic\\",\\"ie\\",\\"im\\",\\"is\\",\\"it\\",\\"je\\",\\"li\\",\\"lt\\",\\"lu\\",\\"lv\\",\\"mf\\",\\"mq\\",\\"mt\\",\\"nc\\",\\"nl\\",\\"no\\",\\"pf\\",\\"pl\\",\\"pm\\",\\"pt\\",\\"re\\",\\"ro\\",\\"se\\",\\"si\\",\\"sj\\",\\"sk\\",\\"tf\\",\\"va\\",\\"wf\\",\\"yt\\"]}}"`;

exports[`Media Asset Page amp Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/persian"`;

exports[`Media Asset Page amp Footer I can see the copyright text 1`] = `"© 2020 بی بی سی. بی بی سی مسئول محتوای سایت های دیگر نیست. سیاست ما درباره لینک دادن به سایت های دیگر."`;

exports[`Media Asset Page amp Header I can see the branding 1`] = `"BBC News, فارسی"`;

exports[`Media Asset Page amp I can see the headline 1`] = `"Trump long headline"`;

exports[`Media Asset Page amp I can see the timestamp 1`] = `"۲۶ بهمن ۱۳۹۷ - ۱۵ فوریه ۲۰۱۹"`;

exports[`Media Asset Page amp Media player A11y I can read the media player title 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/cps/persian/iran-23231114/p01lmkjk/fa/amp"`;

exports[`Media Asset Page amp SEO Canonical link 1`] = `"http://localhost:7080/persian/iran-23231114"`;

exports[`Media Asset Page amp SEO Dir attribute 1`] = `"rtl"`;

exports[`Media Asset Page amp SEO FB admins 1`] = `"100004154058350"`;

exports[`Media Asset Page amp SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Media Asset Page amp SEO Heading level 1 1`] = `"Trump long headline"`;

exports[`Media Asset Page amp SEO Lang attribute 1`] = `"fa"`;

exports[`Media Asset Page amp SEO OG description 1`] = `"Trump summary text"`;

exports[`Media Asset Page amp SEO OG image 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_persian/14DFC/test/_63700558_p01lmkk5.jpg"`;

exports[`Media Asset Page amp SEO OG image alt 1`] = `"Trump getting upset"`;

exports[`Media Asset Page amp SEO OG locale 1`] = `"fa"`;

exports[`Media Asset Page amp SEO OG site name 1`] = `"BBC News فارسی"`;

exports[`Media Asset Page amp SEO OG title 1`] = `"Trump long headline - BBC News فارسی"`;

exports[`Media Asset Page amp SEO OG type 1`] = `"article"`;

exports[`Media Asset Page amp SEO OG url 1`] = `"http://localhost:7080/persian/iran-23231114"`;

exports[`Media Asset Page amp SEO Page title 1`] = `"Trump long headline - BBC News فارسی"`;

exports[`Media Asset Page amp SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Media Asset Page amp SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Media Asset Page amp SEO Twitter creator 1`] = `"@bbcpersian"`;

exports[`Media Asset Page amp SEO Twitter description 1`] = `"Trump summary text"`;

exports[`Media Asset Page amp SEO Twitter image alt 1`] = `"Trump getting upset"`;

exports[`Media Asset Page amp SEO Twitter image src 1`] = `"http://ichef.test.bbci.co.uk/news/1024/branded_persian/14DFC/test/_63700558_p01lmkk5.jpg"`;

exports[`Media Asset Page amp SEO Twitter site 1`] = `"@bbcpersian"`;

exports[`Media Asset Page amp SEO Twitter title 1`] = `"Trump long headline - BBC News فارسی"`;
